docs/user_workloads/liveness_and_readiness_probes.md

```diff
@@ -278,10 +265,10 @@ as `/dev/watchdog`. This device can then be used by the `watchdog`
 binary. For example, if root executes this command inside the VM:
 
 ```bash
-sudo busybox watchdog -t 2000ms -T 4000ms /dev/watchdog
+echo > /dev/watchdog
```

Comment:
this will cause the vm to be terminated after 30 seconds (the default behavior)

Reply:
With my suggested change, this can be rolled back

Reply:
Okay, np, I'm fine with that. Just done it